### PR TITLE
lock fabric loader version in packwiz pack

### PR DESCRIPTION
**Pull request by oatmealine**
want to lock this in right now to avoid dumb inconsistency issues down the line. it has already caused one while testing this!

### CODE REVIEW

**Comment by Patbox** [comment]
0.15.2 released so you might want to bump it

**Comment by deniz-blue** [comment]
yea latest is 0.15.2 👍

File: pack/pack.toml [hunk 1]
```diff
@@ -10,4 +10,4 @@ preserve = false
 
 [versions]
 minecraft = "1.20.4"
-fabric = "latest"
+fabric = "0.15.1"
```
Comment (IThundxr) [inline pack/pack.toml:13]:
```suggestion
fabric = "0.15.2"
```

**Review by IThundxr — APPROVED** [review]

**Comment by deniz-blue** [comment]
wait guys the packwiz pack is auto generated itll just get rewritten

**Pull request merged**